src/app/tests/TestCommandInteraction.cpp

```diff
@@ -577,6 +579,36 @@ void TestCommandInteraction::AddInvokeResponseData(nlTestSuite * apSuite, void *
     }
 }
 
+uint32_t TestCommandInteraction::GetAddResponseDataOverheadSizeForPath(nlTestSuite * apSuite,
+                                                                       const ConcreteCommandPath & aRequestCommandPath,
+                                                                       bool aIsForcedSizeBufferLargerThan255)
+{
+    BasicCommandPathRegistry<4> mBasicCommandPathRegistry;
+    CommandHandler commandHandler(kCommandHandlerTestOnlyMarker, &mockCommandHandlerDelegate, &mBasicCommandPathRegistry);
+    commandHandler.mReserveSpaceForMoreChunkMessages = true;
+    ConcreteCommandPath requestCommandPath1          = { kTestEndpointId, kTestClusterId, kTestCommandIdFillResponseMessage };
+    ConcreteCommandPath requestCommandPath2          = { kTestEndpointId, kTestClusterId, kTestCommandIdCommandSpecificResponse };
+    Optional<uint16_t> commandRef;
+    commandRef.SetValue(1);
+    mBasicCommandPathRegistry.Add(requestCommandPath1, commandRef);
```

Comment:
nit this seems to indicate a bit of an API deficiency ... I would have imagined we would want to support `Add(..., MakeOptional<uint16_t>(1))` Maybe an add with `&&` would work.

Could we maybe NL_TEST_ASSERT CHIP_NO_ERROR here since this returns an error code?